docs/sources/developers/plugins/migration-guide.md

```diff
@@ -33,6 +35,14 @@ This guide helps you identify the steps you need to take based on the Grafana ve
     - [Migrate to data frames](#migrate-to-data-frames)
   - [Troubleshoot plugin migration](#troubleshoot-plugin-migration)
 
+## From version 8.3.x to 8.4.x
+
+This section explains how to migrate Grafana v8.3.x plugins to the updated plugin system available in Grafana v8.4.x. Depending on your plugin, you need to perform one or more of the following steps.
+
+### Value Mapping Editor has been removed from @grafana-ui library
+
+Removed due to being an internal component.
```

Comment:
The text and messaging here needs some :heart: -- lets move any notes to its own PR.  This feels like something that should be as minor as possible since any usage would be pretty exceptional

Reply:
lets move the docs changes to its own PR and address it after doing the same with thersholds